### ISSUE TITLE
Logs file if errors occure during nuxt generate

### ISSUE THREAD

**Issue by Patrick-ISKN** [nuxt/nuxt.js#5927]
### What problem does this feature solve?

Hey there, I'd like to know if a feature to get logs during a nuxt generation of pages, like a log file, could be done. I have a really generic error in my project (I use Nuxt and Storyblok) when I generate my project, see the image below 
![Capture.PNG](https://cmty.s3.eu-west-1.amazonaws.com/71995937/a41840a0-8de9-11e9-bb9f-2dc69d4072b3.PNG)
And there's no way to get more information about those type of errors ,and I found nothing through the docs and the web... So could it be possible to get more informations for these generical errors ? Are they critical ? Could there be a way to log those errors ?
I already asked this as a question on the cmty app, but got no answer
Thanks! 

### What does the proposed changes look like?

Get more informations on every type of errors during nuxt generate

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c9356">#c9356</a>)</em></sub></div>

**Comment by pimlie** [comment]
You can [hook](https://nuxtjs.org/api/configuration-hooks#the-hooks-property) into [either](https://nuxtjs.org/api/internals-generator#hooks)
- `generate:routeCreated` to receive an error immediately when it occured or
- `generate:done`, this is called after all routes have been generated and receives a list of all errors

If you use these hooks to log the errors with consola, then you can eg add stdin/stderr to a log file or you can add a consola reporter which logs to a file.

**Comment by Patrick-ISKN** [comment]
Hey @pimlie , thanks to your steps I catched the error I was looking for, and it is now working fine during generation, thanks a lot! 
The error I catched with the hook `generate:routeCreated` is the following: `HTML minification failed. Make sure the route generates valid HTML.`  
And the solution for my original error is [this way](https://github.com/nuxt-community/starter-template/issues/28#issuecomment-324858498), as simple as it is...
Thanks again!
